js/axios/api-keys/fetchKeys.js

```diff
@@ -40,23 +43,29 @@ const filters = "";
 (async () => {
   // fetch all keys
   const apiKeys = await fetchApiKeys(filters);
-  console.log('All Keys', apiKeys);
+  console.log("All Keys", apiKeys);
 
-  // get keys with a specific name
-  const nameFilteredKeys = await fetchApiKeys(getFiltersQuery({
-    name: "new api key",
-  }));
-  console.log('Keys filtered using name', nameFilteredKeys);
+  // fetch keys with a specific name (1st one from the list returned previously)
+  const nameFilteredKeys = await fetchApiKeys(
+    getFiltersQuery({
+      name: apiKeys[0].name,
```

Comment:
This can cause runtime error if apiKeys is empty `[]` (trying to access the first element of an empty list)

it would be better to update it something that like 
```
name: apiKeys.length === 0 ? 'YOUR_NAME_FILTER' : apiKeys[0].name
```

Reply:
Pushed your suggestion